Package.swift

```diff
@@ -0,0 +1,100 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Amplify",
+    platforms: [.iOS(.v11)],
+    products: [
+        .library(
+            name: "Amplify",
+            targets: ["Amplify"]),
+        
+        .library(name: "AWSPluginsCore",
+                 targets: ["AWSPluginsCore"]),
+        
+        .library(name: "AWSAPIPlugin",
+                 targets: ["AWSAPIPlugin"]),
+        
+        .library(name: "AWSCognitoAuthPlugin",
+                 targets: ["AWSCognitoAuthPlugin"]),
+        
+        .library(name: "AWSDataStorePlugin",
+                 targets: ["AWSDataStorePlugin"]),
+        
+        .library(name: "AWSPinpointAnalyticsPlugin",
+                 targets: ["AWSPinpointAnalyticsPlugin"]),
+        
+        .library(name: "AWSS3StoragePlugin",
+                 targets: ["AWSS3StoragePlugin"]),
+        
+    ],
+    dependencies: [
+        .package(name: "AWSiOSSDKV2", url: "https://github.com/aws-amplify/aws-sdk-ios-spm.git", from: "2.23.3"),
```

Comment:
SDK doesn't follow semantic versioning
```suggestion
        .package(name: "AWSiOSSDKV2", url: "https://github.com/aws-amplify/aws-sdk-ios-spm.git", .upToNextMinor(from: "2.23.3")),
```